packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -9890,7 +9864,7 @@ Generated by [AVA](https://ava.li).
     ────────────────────────────────────────────────────────────────␊
     ␊
       Error message␊
-      Error: Deleting "netlifyConfig.build.ignore" is not allowed.␊
+      Error: Setting "netlifyConfig.build.command" to undefined is not allowed.␊
```

Comment:
We are detecting property deletion with a different function now, which explains the difference of error message.